### PR TITLE
Karel/layout - created homepage and study options based on materials in figma

### PR DESCRIPTION
**Pull request by KarelGeyer**
Couple of components were created:

**Homepage** 
folder 
- can be found -> frontend -> pages -> Homepage (localhost:3000/homepage)
- needs to be redesigned to match with version 3.0 (will be done in next few days)
- added responsivity (only phone version) 
- all styled using Styled components, 
-  based on version 0.5, for now, its components are:
1.  DropDowns.tsx - basic styled inputs to choose your education, course and additional qualification, under inputs there is a search button.
2.  Headline.tsx - Headline and Subheadline

**StudyOptions*
- can be found -> frontend -> pages -> StudyOptions (localhost:3000/studyoptions)
- design based on version 3.0
- added responsivity (only phone version) 
- all styled using Styled components, Material UI for icons
- its components are: 
1. Heading.tsx - headings, title, qualification, some text, filter button
2. Selection.tsx - selection based on Tile, Qualification, CŽV
3. Results.tsx - containers for each university

**Footer**
- Footer can be found in - folder layout -> footer -> Footer.tsx
- once again only used Styled Components
- its responsive (only phone version)
- imported in Layout.tsx bellow MaxWidth component

also installed material UI Core and Icons

### CODE REVIEW

**Comment by jpetera54** [comment]
Za mě ok, klidně mergnout.
Super práce Karle

File: frontend/layout/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    display: flex;
+    @media (max-width: 670px) {
+        height: 400px;
+        flex-direction: column;
+        margin-left:0%;
+    }
+    
+`
+/**Fonts */
+const FooterH3 = styled.h3 `
```
Comment (steveheaven) [inline frontend/layout/footer/Footer.tsx:41]: všechny "há" elementy(h1, h2,...) bych v moderním vývoji moc nepoužíval, lepší je div nebo span nastylovanej podle globálních pravidel v layout/Theme.ts...há trojky mají tendenci se za určitých okolností chovat na více prohlížečích různě(třeba v Chromu to jede v pohodě, ale v Mozille máš font defaultní, apod...)

**Review by steveheaven — CHANGES_REQUESTED** [review]

File: frontend/layout/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+import styled from "styled-components"
+
```
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/layout/footer/Footer.tsx:3]: Sám tam nic neplánuju, nebyl jsem si ale jsitej jestli tam nebude nutné nějakou komponentu přidat. Složku teda mažu.

File: frontend/layout/footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    padding-left: 50px;
+    padding-bottom: 10px;
+    @media (max-width: 670px) {
+        padding-left: 15px;
+        padding-bottom: 0px;
+    }
+`
+/**Logo */
+const Logocontainer = styled.div `
+
```
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/layout/footer/Footer.tsx:63]: předělám :)

File: frontend/pages/homepage/DropDowns.tsx [hunk 4]
```diff
@@ -0,0 +1,197 @@
+import React from 'react'
+import styled from "styled-components"
+
```
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/pages/homepage/DropDowns.tsx:3]: ok do dalšího meetingu upravím.

File: frontend/layout/footer/Footer.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/layout/footer/Footer.tsx:3]: pozn. Done

File: frontend/layout/footer/Footer.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/layout/footer/Footer.tsx:63]: pozn. done

File: frontend/pages/studyoptions/components/Results.tsx [hunk 5]
```diff
@@ -0,0 +1,188 @@
+import React, {useState} from 'react'
+import styled from 'styled-components';
+import SchoolIcon from '@material-ui/icons/School';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import WatchLaterIcon from '@material-ui/icons/WatchLater';
+import RoomIcon from '@material-ui/icons/Room';
+import { BrokenImage } from '@material-ui/icons';
+
```
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/pages/studyoptions/components/Results.tsx:8]: pozn. done

File: frontend/layout/footer/Footer.tsx [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+ 
+    max-width: 640px;
+    height: 70px;
+    text-align: center;
+    @media (max-width: 670px) {
+        width: 200px;
+    }
+`
+const BlankDivContainer = styled.div `
+
```
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/layout/footer/Footer.tsx:89]: pozn. done

File: frontend/pages/homepage/index.tsx [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import React,{useState} from 'react'
+import Dropdowns from './DropDowns'
+import Headline from './Headline'
+
+
+
+
+const Homepage = () => {
+
```
[the remark this answers is not in the document]
Comment (KarelGeyer) [reply frontend/pages/homepage/index.tsx:9]: pozn. done

**Pull request merged**